lib/models/settings_model.dart

```diff
@@ -8,23 +8,24 @@ import 'package:api_client/models/enums/default_timer_enum.dart';
 import 'package:api_client/models/model.dart';
 import 'package:api_client/models/enums/orientation_enum.dart';
 import 'package:api_client/models/weekday_color_model.dart';
+import 'enums/weekday_enum.dart';
 
 /// A model used to store settings values
 class SettingsModel implements Model {
   /// Constructor
   SettingsModel(
       {@required this.orientation,
-        @required this.completeMark,
-        @required this.cancelMark,
-        @required this.defaultTimer,
-        this.timerSeconds,
-        this.activitiesCount,
-        @required this.theme,
-        this.nrOfDaysToDisplay,
-        this.lockTimerControl,
-        this.pictogramText,
-        this.greyscale,
-        this.weekDayColors});
+      @required this.completeMark,
```

Comment:
Is the indentation supposed to be changed here?

Reply:
Yes